domains/slyness.json

```diff
@@ -0,0 +1,14 @@
+{
+    "owner": {
+        "description": "Personal Website for Slyness",
+        "username": "slyness6",
+        "email": "slynesss@proton.me"
+        "discord": "lbt6"
+    },
+    "record": {
+    "NS": [
+	"jake.ns.cloudflare.com",
+	"stephane.ns.cloudflare.com"
+    ]
```

Comment:
Can you give a reason as to why you need NS records?

Reply:
> Can you give a reason as to why you need NS records?
bc ill be using it as a personal website and i need the ns records

Reply:
bc i need to link it wit cloudflare

Reply:
Not a valid reason.